daisy_workflows/linux_common/utils.py

```diff
@@ -65,6 +65,11 @@ def PipInstall(package_list):
   return Execute(['pip', 'install', '-U'] + package_list)
 
 
+def DpkgInstall(package_list):
+  """Install a package from .deb file via dpkg."""
+  return Execute(['dpkg', '-i'] + package_list)
```

Comment:
Would it be better to use apt-get here... apt-get install ./package.deb so that dependencies are taken into account.

Reply:
True- even better :)